keylime/policy/create_runtime_policy.py

```diff
@@ -43,6 +43,9 @@
 IGNORED_KEYRINGS: List[str] = []
 DEFAULT_FILE_DIGEST_ALGORITHM = algorithms.Hash.SHA256
 
+# This is to follow IMA default
+# See in keylime/ima/ima.py
+DEFAULT_IMA_TEMPLATE_DIGEST_ALGORITHM = algorithms.Hash.SHA1
```

Comment:
Yes, I think this is still actually hard-coded in the kernel anyway.

Reply:
Unfortunately, it is.